src/backend/columnar/columnar_tableam.c

```diff
@@ -67,6 +71,13 @@
 #define VACUUM_TRUNCATE_LOCK_WAIT_INTERVAL 50       /* ms */
 #define VACUUM_TRUNCATE_LOCK_TIMEOUT 4500               /* ms */
 
+#define COLUMNAR_PG_COMPRESSION_NOTICE_MSG \
+	"since columnar tableAM already compresses columns in large chunks, " \
+	"using built-in postgres compression might not bring additional " \
```

Comment:
Is it `might not` or `certainly not`? My understanding is that because we store data in a different place, column compression would certainly not help at this point?

If that's the case, I was thinking something like:

```
Columnar tables already compress data with a different mechanism, ignoring Postgres' column compression method.
```

Reply:
Right, actually it should be "certainly not".
Btw, we don't actually ignore Postgres' compression but just inform users. So, what about the following:

```c
"since columnar tableAM already compresses columns in large chunks, " \
"using built-in postgres compression will not bring additional " \
"compression benefits, see \"alter_columnar_table_set\" function to "\
"use the compression functionality provided by columnarAM"
```